src/System.Drawing.Common/tests/mono/System.Drawing.Drawing2D/GraphicsPathIteratorTest.cs

```diff
@@ -0,0 +1,170 @@
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Security.Permissions;
+using Xunit;
+
+namespace MonoTests.System.Drawing.Drawing2D
+{
+
+    public class GraphicsPathIteratorTest
+    {
+
+        private PointF[] pts_2f = new PointF[2] { new PointF(1, 2), new PointF(20, 30) };
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Ctor_Null()
+        {
+            using (GraphicsPathIterator gpi = new GraphicsPathIterator(null))
+            {
+                Assert.Equal(0, gpi.Count);
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void NextMarker_Null()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.AddLines(pts_2f);
+                using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+                {
+                    Assert.Equal(0, gpi.NextMarker(null));
+                }
+            }
+        }
+
+        [ActiveIssue(20844)]
+        public void NextSubpath_Null()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.AddLines(pts_2f);
+                using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+                {
+                    bool closed;
+                    Assert.Equal(0, gpi.NextSubpath(null, out closed));
+                    Assert.True(closed);
+                }
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void CopyData_NullPoints()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.AddLines(pts_2f);
+                using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+                {
+                    PointF[] points = null;
+                    byte[] types = new byte[1];
+                    Assert.Throws<NullReferenceException>(() => gpi.CopyData(ref points, ref types, 0, 1));
+                }
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void CopyData_NullTypes()
```

Comment:
This is basically covered by an existing test. https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Common/tests/Drawing2D/GraphicsPathIteratorTests.cs#L351